docker/core/scripts/update_index.rb

```diff
@@ -0,0 +1,6 @@
+require "json"
```

Comment:
If this file is needed then it should be copied into the container by the `Dockerfile`, not just copied in the repo (because then people will forget to update this file when updating the original one)

Reply:
You are copying files from the right place now but you still haven't deleted them from the PR